GitUI/CommandsDialogs/BrowseDialog/FormRecentReposSettings.cs

```diff
@@ -163,7 +181,6 @@ private void comboMinWidthEdit_ValueChanged(object sender, EventArgs e)
                 AllRecentLB.BeginUpdate();
 
                 SetComboWidth();
-                RefreshRepos();
             }
             finally
             {
```

Comment:
```
PinnedLB.BeginUpdate();
AllRecentLB.BeginUpdate();
```
seems to be not necessary here any longer.
Though it should be added to `RefreshRepos()`.

Reply:
done (With a little bonus: improvement over the bad UX of the "Combobox minimum width" that I had to use for the 1st time to test this piece of code)